modules/change_status.js

```diff
@@ -0,0 +1,39 @@
+import ToDoController from './todo-controller';
+
+const addCheckBoxListeners = () => {
+  const checkBoxes = document.querySelectorAll('.item-check');
+  checkBoxes.forEach((checkBox) => {
+    if (Number(checkBox.classList[1]) >= ToDoController.todos.length) {
+      checkBox.addEventListener('click', (event) => {
+        if (event.target.checked) {
+          event.target.nextElementSibling.style.textDecoration = 'line-through';
+          event.target.nextElementSibling.style.color = 'gray';
+        } else {
+          event.target.nextElementSibling.style.textDecoration = 'none';
+          event.target.nextElementSibling.style.color = 'black';
+        }
+        ToDoController.changeStatus(Number(event.target.classList[1]));
+        // return;
```

Comment:
- [ ] [OPTIONAL] PLEASE try to remove this unused code

Reply:
Thanks very much for the note @youmari